pkg/manager/manager_table.go

```diff
@@ -149,9 +134,37 @@ func (sm *Manager) startTableMode(id string) error {
 			err = sm.svcProcessor.ServicesWatcher(ctx, sm.svcProcessor.SyncServices)
 			if err != nil {
 				log.Error("Cannot watch services", "err", err)
+			}else{
+				log.Debug("watching services")
 			}
 		}
 	}
+	if sm.config.EnableControlPlane {
```

Comment:
can we move this part into a separate PR?
I would like to merge it and it does not change behaviour that much.

Reply:
Please ensure go fmt run.